### ISSUE TITLE
ImunAPIWrapper

### ISSUE THREAD

**Issue by aryansid** [microsoft/MM-REACT#51]
Hey, 

It's confusing where we can get access to ImunAPIWrapper. You seem to be importing from langchain.utilities, but ImunAPIWrapper does not exist. Tried having a look through the code, but couldn't find anything promising. 

Please let me know!

**Comment by jedetiney** [comment]
Have the same problem. Tried finding any related documentation on Langchain as well as Azure, none found

**Comment by aryansid** [comment]
Yeah, I tried searching for ages. Found nothing, so I just adapted the code for my needs.

**Comment by gopalchettri** [comment]
> Yeah, I tried searching for ages. Found nothing, so I just adapted the code for my needs.
@aryansid could you please elaborate how you did that?

**Comment by jun0wanan** [comment]
> Yeah, I tried searching for ages. Found nothing, so I just adapted the code for my needs.

Can you give us your code? thx...
